### ISSUE TITLE
Bug in pooling.py

### ISSUE THREAD

**Issue by Sargul** [cannatag/ldap3#132]
pooling.py
Line 124 - 127:
```python
                if server[0].check_availability():
                    # returns a random active server in the pool
                    server[2] = True
                    return self.servers.index(server)[0]
```

```pythonself.servers.index``` returns int and access to element 0 will fail with 
```TypeError: 'int' object has no attribute '__getitem__'```

**Comment by Sargul** [comment]
I've created a fork where I fixed the issue. I've allready tested this with multiple servers in the ServerPool.

**Comment by cannatag** [comment]
THanks Sargul, I will check and include it in the next

**Comment by cannatag** [comment]
Released in 1.0.1
